### PR TITLE
write_tsdb plugin: Prevent DNS flooding on connection error.

### PR DESCRIPTION
**Pull request by octo**

### CODE REVIEW

File: src/collectd.conf.pod [hunk 1]
```diff
@@ -8075,7 +8077,34 @@ Synopsis:
...
+Global directives are:
+
+=over 4
+
+=item B<DNS_Cache_TTL> I<ttl>
+
+=item B<DNS_Random_Cache_TTL> I<ttl>
+
+When Collectd connects to a TSDB node, it will request the DNS. This can become
+a problem is the TSDN node is unavailable or badly configured because Collected
```
Comment (kev009) [inline src/collectd.conf.pod]: Collected

File: src/collectd.conf.pod [hunk 2]
```diff
@@ -8083,24 +8083,25 @@ Global directives are:
...
 
-=item B<DNS_Random_Cache_TTL> I<ttl>
+=item B<ResolveJitter> I<seconds>
 
-When Collectd connects to a TSDB node, it will request the DNS. This can become
-a problem is the TSDN node is unavailable or badly configured because Collected
-will request DNS in order to reconnect for every metric, which can flood your DNS.
-So you can cache the last value for C<ttl> seconds (default: 600s e.g; 10 min).
+When I<collectd> connects to a TSDB node, it will request the DNS. This can
+become a problem if the TSDN node is unavailable or badly configured because
```
Comment (kev009) [inline src/collectd.conf.pod]: TSDN->TSDB

File: src/collectd.conf.pod [hunk 3]
```diff
@@ -8083,24 +8083,25 @@ Global directives are:
...
+collectd will request DNS in order to reconnect for every metric, which can
+flood your DNS. So you can cache the last value for I<ResolveInterval> seconds
+(default: B<600>, i.e. 10 minutes).
 
-You can also define a random ttl. This prevents all your Collectd servers to
-request the DNS at the same time when the connection fails. Default value is
-15 * the write_tsdb interval (or the global interval if write_tsdb interval is not
-defined).
+You can also define a jitter, a random interval to wait in addition to
```
Comment (kev009) [inline src/collectd.conf.pod]: I would instead suggest calling this a "splay" in both config param and documentation, but just a minor nit.

**Comment by kev009** [comment]
Since it got buried in the commit diff, I suggest splay in place of jitter for describing the option but otherwise LGTM

**Comment by octo** [comment]
Personally, I haven't heard "splay" before but am quite familiar with the "jitter" term. Anybody else have any opinion? @rubenk? @tokkee?

**Comment by rubenk** [comment]
As a non-native english speaker I've had to look up splay, so jitter sounds better.

**Comment by kev009** [comment]
I stole splay from cfengine, which seems to have spread it to Chef and Salt

**Comment by octo** [comment]
Alright, we're going to stick with "jitter" then.

**Pull request merged**